From: Colin Ian King <colin.i.king@googlemail.com>
Date: Sat, 23 Oct 2021 12:52:09 +0100
Subject: optee: Fix spelling mistake "reclain" -> "reclaim"
Git-commit: 1b73a9e4986a4e9065bacf1e5ab2dfda17b54161
Patch-mainline: v5.16-rc1
References: jsc#SLE-21844

There are spelling mistakes in pr_err error messages. Fix them.

Fixes: 4615e5a34b95 ("optee: add FF-A support")
Signed-off-by: Colin Ian King <colin.i.king@gmail.com>
Reviewed-by: Sumit Garg <sumit.garg@linaro.org>
[jw: added a fixes]
Signed-off-by: Jens Wiklander <jens.wiklander@linaro.org>
Signed-off-by: Matthias Brugger <mbrugger@suse.com>
---
 drivers/tee/optee/ffa_abi.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/tee/optee/ffa_abi.c b/drivers/tee/optee/ffa_abi.c
index 6defd1ec982a..45424824e0f9 100644
--- a/drivers/tee/optee/ffa_abi.c
+++ b/drivers/tee/optee/ffa_abi.c
@@ -333,7 +333,7 @@ static int optee_ffa_shm_unregister(struct tee_context *ctx,
 
 	rc = ffa_ops->memory_reclaim(global_handle, 0);
 	if (rc)
-		pr_err("mem_reclain: 0x%llx %d", global_handle, rc);
+		pr_err("mem_reclaim: 0x%llx %d", global_handle, rc);
 
 	return rc;
 }
@@ -355,7 +355,7 @@ static int optee_ffa_shm_unregister_supp(struct tee_context *ctx,
 	optee_shm_rem_ffa_handle(optee, global_handle);
 	rc = ffa_ops->memory_reclaim(global_handle, 0);
 	if (rc)
-		pr_err("mem_reclain: 0x%llx %d", global_handle, rc);
+		pr_err("mem_reclaim: 0x%llx %d", global_handle, rc);
 
 	shm->sec_world_id = 0;
 
-- 
2.33.1

